libcnb/src/layer/test.rs

```diff
@@ -922,10 +922,8 @@ fn build_context(temp_dir: &TempDir) -> BuildContext<TestBuildpack> {
 }
 
 fn random_layer_name() -> LayerName {
-    rand::thread_rng()
-        .sample_iter(rand::distributions::Alphanumeric)
+    repeat_with(fastrand::lowercase)
```

Comment:
FWIW: this changes the test slightly since the names are now only lowercased.

```suggestion
    repeat_with(fastrand::alphanumeric)
```

Reply:
Yeah I noticed that, but had thought it didn't matter, since otherwise we'd have explicit tests for layer casing, rather than it be non-deterministic? However happy to change it back for parity.